src/frontend/screens/WineManager/index.css

```diff
@@ -1,4 +1,7 @@
 .wineManager {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
the loading element in this page was always adding vertical scroll, using flex to avoid that